pkgs/games/minecraft-server/default.nix

```
@@ -26,9 +27,13 @@ stdenv.mkDerivation {
 
   meta = {
```

since we are doing some linting
```suggestion
  meta = with stdenv.lib; {
```